source/_integrations/matter.markdown

```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
 
 Running Matter on a Home Assistant Core installation is not supported.
 
-## Adding Matter devices to Home Assistant
+## Adding a Matter device to Home Assistant
 
 Each Matter network is called a fabric. Each home automation controller that controls Matter devices has its own "fabric". You can add devices directly to the fabric of your Home Assistant instance, or share them from another fabric (ie Google, Apple) to Home Assistant's fabric. We're going to explore all these options below.
 
-### Add a device using the iOS Companion app
+### Prerequisites
+
+- On the device packaging, check for the Matter logo and for either the Wi-Fi or the Thread logo.
+- Check if the QR code is only on the packaging or if it is also on the device.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store it in a save place.
+  - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
+- In Home Assistant, have the Matter integration installed.
+  - Go to {% my integrations title="**Settings** > **Devices & services**" %}.
+  - Add the **Matter (BETA)** integration.
+  - When prompted to **Select the connection method**:
+    - If you run Home Assistant OS in a regular setup: select **Submit**.
+      - This will install the official Matter server add-on.
+    - If you are already running the Matter server in another add-on, a custom container, or natively:
+      - Deselect the checkbox, then Select **Submit**.
+      - In the next step, provide the URL to your Matter server.
+- Have either an Android or iPhone ready and Bluetooth enabled. For information why Bluetooth is required, refer to the section on [Bluetooth used during commissioning](#bluetooth-used-during-commissioning):
+  - Android:
+    - Have an Android phone (a full Android, not F-Droid).
+    - Have the Home Assistant Companion app installed.
+    - Have Google Home app installed on the Android.
+    - We are not going to add the new device to Google Home. The app is needed because Google included the Matter SDK there.
+    - If you are using Thread: Make sure there is a Thread border router device (Nest Hub v2 or Nest Wi-Fi Pro) added to Home Assistant.
+  - iPhone
+    - Version 16 or higher
+    - If you are using Thread: Make sure there is a Thread border router device (HomePod Mini or V2, Apple TV 4K) added to Home Assistant.
+      - For instruction on how to add these devices to Home Assistant, refer to the instructions on [adding a HomeKit device](/integrations/homekit_controller/#adding-a-homekit-device-via-ethernet-or-wi-fi).
+
```

Comment:
```suggestion
```

Reply:
Maybe we should link here to the Thread documentation or any other documentation how thread border routers work and how we utilize them ?

Reply:
Agreed. I was thinking about that, too. Can we add the links once we double-checked/updated the Thread docs? I want to be sure they are updated before linking there. Making a note for another PR.